koku/api/report/gcp/query_handler.py

```diff
@@ -3,6 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 """GCP Query Handling for Reports."""
+from __future__ import annotations
```

Comment:
Was this a vs code auto-import? Guessing we don't want this.

Reply:
Most definitely was an auto-import 👍  I removed it - thank you!